hpx/parallel/algorithms/transform.hpp

```diff
@@ -129,8 +129,10 @@ namespace hpx { namespace parallel { inline namespace v1
             operator()(Iter part_begin, std::size_t part_size,
                 std::size_t /*part_index*/)
             {
+#if !defined(__NVCC__) && !defined(__CUDACC__)
```

Comment:
As this might cause problems in other places of the code, wouldn't it be better to provide an empty implementation of `annotate_function` for this configuration?